docker-image/r/spark2.2.0/gpu/Dockerfile

```diff
@@ -0,0 +1,142 @@
+# Ubuntu 16.04 (Xenial)
+FROM aztk/gpu:spark2.2.0
+
+# modify these ARGs on build time to specify your desired versions of Spark/Hadoop
+ARG R_VERSION=3.4.1
+ARG RSTUDIO_SERVER_VERSION=1.1.383
+ARG USER_PYTHON_VERSION=3.5.4
+ARG CNTK_GPU_VERSION=https://cntk.ai/PythonWheel/GPU/cntk-2.3.1-cp35-cp35m-linux_x86_64.whl
+ARG BUILD_DATE
+
+# set env vars
+ENV DEBIAN_FRONTEND noninteractive
+ENV BUILD_DATE ${BUILD_DATE:-}
+ENV RSTUDIO_SERVER_VERSION $RSTUDIO_SERVER_VERSION
+ENV R_VERSION $R_VERSION
+
+RUN  useradd -m -d /home/rstudio rstudio -G sudo,staff \
+  && echo rstudio:rstudio | chpasswd \
+  && chmod -R 777 /home/rstudio \
+  && chmod -R 777 //.pyenv/
+
+# Setting up rstudio user with Tensorflow and CNTK
+USER rstudio
+RUN  echo "PATH='"$PATH"'" > /home/rstudio/.Renviron \
+  && pip3 install \
+    $CNTK_GPU_VERSION \
```

Comment:
any reason we're not using the same variable definition for tensorflow and keras?